src/app/in-out-transform.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'inOutTransform',
+})
+export class InOutTransformPipe implements PipeTransform {
+  transform(value: number, ...args: unknown[]): unknown {
+    if (value === 0) {
+      return 'Wyjście';
+    } else {
+      return 'Wejście';
+    }
```

Comment:
Ten warunek można zapisać prościej:

```suggestion
    if (!value) {
      return 'Wyjście';
    }
     return 'Wejście';
```

Czyli jeśli wartość jest "falsy" (0 jest falsy) to drukuj 'wyjście' w innym przypadku  drukuj 'wejście'

Reply:
Dla mnie zapis z zerem jest bardziej czytelny ze względu na to, że w taki sposób są zapisane wejścia i wyjścia w bazie danych.